Backend/app/rider-platform/rider-app/Main/src/Domain/Action/UI/Search/Common.hs

```diff
@@ -65,7 +65,8 @@ buildSearchRequest person pickup mbDrop mbMaxDistance mbDistance now bundleVersi
         device = device,
         bundleVersion = bundleVersion,
         clientVersion = clientVersion,
-        language = person.language
+        language = person.language,
+        customerExtraFee = Nothing
```

Comment:
If it always Nothing, why do you need it at all?

Reply:
It is always nothing if user gives no tip, otherwise if he gives tip then it will be updated